docs/csharp/programming-guide/classes-and-structs/how-to-implement-a-lightweight-class-with-auto-implemented-properties.md

```diff
@@ -27,7 +50,7 @@ The following example shows two ways to implement an immutable class that has au
 // constructor to initialize its properties.
 class Contact
 {
-    // Read-only properties.
+    // Read-only property.
     public string Name { get; }
     public string Address { get; private set; }
```

Comment:
I don't think the "Address" property is considered as read-only. Only "Name" is read-only. So, I changed "Read-only properties" to "Read-only property".

Reply:
In this instance, we need to clarify the subtlety:

From the perspective of the public interface, `Address` is a read-only property. In contrast, from the perspective of immutability and members of the type, `Address` is not read-only.

Reply:
I think the best way to highlight this distinction is to refer to one as "read-only" property, and the other as a "read-write property with a private set accessor"

It is more wordy, but it is consistent with the spec language.